test/house_test.rb

```diff
@@ -0,0 +1,86 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists_and_has_attributes
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_instance_of House, house
+    assert_equal 400000, house.price
+    assert_equal "123 sugar lane", house.address
+    assert_equal [], house.rooms
+  end
+
+  def test_it_can_return_details
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal {"price"=>400000, "address"=>"123 sugar lane"}, house.details
+  end
+
+  def test_it_can_have_rooms
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+
+    assert_instance_of Room, room_1
+
+    room_2 = Room.new(:bedroom, 11, '15')
+
+    assert_instance_of Room, room_2
```

Comment:
It doesn't hurt to do two different rooms, but unless there's something substantially different about these two rooms, it probably doesn't do you any good, either, so we can just test one.